assets/js/player.js

```diff
@@ -59,6 +49,38 @@ videojs.Vhs.xhr.beforeRequest = function(options) {
 
 var player = videojs('player', options);
 
+/**
+ * Function for add time argument to url
+ * @param {String} url 
+ * @returns urlWithTimeArg
+ */
+function addCurrentTimeToURL(url) {
+    var urlUsed = new URL(url);
+    urlUsed.searchParams.delete('start');
```

Comment:
Can those be inlined like so?
`var urlUsed = new URL(url).searchParams.delete('start');`

Reply:
Yes, it wouldn't change much in the end

Reply:
If indeed, I can not reduce it otherwise the variable `urlUsed` receives the result `void` of the method `.delete('start')`